docs/user_guide/mitigation/postprocessing.rst

```diff
@@ -0,0 +1,448 @@
+We can relax this a little bit since in many cases one lies strictly above
+the other, just like in this example.
+Imagine the diagonal of the ROC plot from :math:`(0,0)` to :math:`(1,1)`.
+It is usually not considered as useful since we could achieve the same result
+with random classifiers with probabilities to get label 1 ranging from 0 to 1.
+If we have one curve strictly below the other, we can still get their true and
+false positive rates to match by "pulling" the higher one towards the
+diagonal. In reality we consider the overlap, so only the points on the lower
+curve are available to us. From those points we choose the one with the best
+value in terms of the overall objective (not per group).
+Stepping below the Pareto curve for any sensitive feature group means
+introducing a probability (:code:`p_ignore`) that we need to toss a biased
+coin whose bias is indicated by :code:`prediction_constant`.
+
+Mathematically, the following formula represents this interpretation of the
+chart, where :math:`c` represents the `prediction_constant` from the printed
+dictionary above.
+
+.. math::
+
+    p_{\text{ignore}} \cdot c + (1-p_{\text{ignore}}) \cdot \left(p_0 \cdot \text{operation}_0(\text{score}) + p_1 \cdot \text{operation}_1(\text{score})\right)
+
+In other words, we create a linear combination of the binary thresholding rule
+outputs. :math:`p_0` and :math:`p_1` indicate how close we are to the points
+representing the two thresholding rules, and :math:`p_0+p_1=1`.
+:math:`p_{\text{ignore}}` is :math:`0` if the curve does not need to be "pulled"
+towards the diagonal, i.e., when the selected solution lies on the curve
+itself as opposed to below.
+Above, the curve for "Asian" is strictly above the other curves, so
+:math:`p_{\text{ignore}}` is highest for "Asian" and lower for other groups.
+
+Importantly, :math:`p_{\text{ignore}}` is only required for equalized odds.
+We want to stress on the fact that it is crucial to understand the
+implications of such a result.
+Predictions of the resulting :class:`ThresholdOptimizer` model are determined
+randomly based on the probabilities to predict label 1 that the formula
+produces.
+
+In some cases, one of the thresholds are defined to always be true or false
+(e.g., `>-infty` or `<infty`) resulting in base probabilities for getting a 0
+or 1 regardless of the features.
+In our example above, this might mean that members of one group might have a
+base probability of being predicted to be readmitted that is higher than
+the base probability for members of another group.
+We want to emphasize that a non-zero probability to get label 1 is not
+acceptable in some application contexts, so it needs to be carefully evaluated
+whether :class:`ThresholdOptimizer` should be used.
+
+Due to the separate thresholding rules per sensitive feature value, one might
+argue that this constitutes
+`disparate treatment <https://en.wikipedia.org/wiki/Disparate_treatment>`_ in

```

Comment:
under US anti-discrimination law or direct discrimination under EU non-discrimination law (which tragically does not have it's own Wikipedia page)